meta-phosphor/conf/machine/include/sample.inc

```diff
@@ -24,7 +24,7 @@ VIRTUAL-RUNTIME_obmc-phosphor-ipmi-parsers = " \
        "
 PREFERRED_PROVIDER_virtual/obmc-phosphor-host-ipmi-hw = "host-ipmi-hw-example"
 PREFERRED_PROVIDER_virtual/obmc-phosphor-chassis-mgmt = "obmc-phosphor-chassisd"
-PREFERRED_PROVIDER_virtual/obmc-phosphor-event-mgmt = "obmc-phosphor-eventd"
+PREFERRED_PROVIDER_virtual/obmc-phosphor-event-mgmt = "obmc-phosphor-event"
```

Comment:
There is no longer a package called 'eventd'.  @causten seems to have removed it when he added event, or at least that is the best I can tell.  I don't know how this was still compiling on Jenkins but when I did a clean build with yocto 2.1 it started failing.